modules/servers/upCloudVps/lib/configOptions.php

```diff
@@ -78,7 +71,6 @@ private function createGlobalConfigurableOptions($product)
         $this->createLocationFields($groupId, $zones, $currencyId);
         $this->createTemplateFields($groupId, $pomTemplates, $currencyId, $currencyCode);
         $this->createBackupFields($groupId, $this->getBackups());
-        $this->createIPFields($groupId, $this->getNetworking());

```

Comment:
After removing this, the `createIPFields` function is no longer used, let's remove the function itself too (line 87 below).